### PR TITLE
feat: Update dependencies

### PR DESCRIPTION
**Pull request by jonas-martinez**
Closes #

## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I did not include breaking changes
- [x] I made my own code-review before requesting one

### I included unit tests that cover my changes
  - [ ] 👍 yes
  - [x] 🙅 no, because they aren't needed
  - [ ] 🙋 no, because I need help

### I added/updated the documentation about my changes

- [ ] 📜 README.md
- [ ] 📕 docs/*.md
- [ ] 📓 docs.lenra.io
- [x] 🙅 no documentation needed

### CODE REVIEW

File: client/pubspec.yaml [hunk 1]
```diff
@@ -39,11 +39,11 @@ dependencies:
   client_common:
     git:
       url: git@github.com:lenra-io/client-common.git
-      ref: v1.0.0-beta.40
+      ref: v1.0.0-beta.41
   lenra_ui_runner:
     git:
       url: git@github.com:lenra-io/lenra_ui_runner.git
-      ref: v1.0.0-alpha.5
+      ref: v1.0.0-alpha.1
```
Comment (pichoemr) [inline client/pubspec.yaml:46]: Why did you downgrade version ?

Comment (jonas-martinez) [reply client/pubspec.yaml:46]: There was a problem with semantic release on lenra_ui runner ^^ We had to remove all the alpha tags and start from zero.

File: client/pubspec.lock [hunk 2]
```diff
@@ -403,8 +403,8 @@ packages:
     dependency: "direct main"
     description:
       path: "."
-      ref: "v1.0.0-alpha.5"
```
Comment (taorepoara) [inline client/pubspec.lock:406]: @jonas-martinez You've downgraded the version here ?

**Review by taorepoara — APPROVED** [review]

**Pull request merged**